modules/dynamic-operator/plugins/docker-compose/plugin/compose/objects.go

```diff
@@ -32,11 +33,6 @@ const (
 )
 
 var (
-	serviceAccountGVK = schema.GroupVersionKind{
```

Comment:
Why service account is deleted?

Reply:
Pods will be using `default` ServiceAccount when it is not set, it was in task spec